R/proj.R

```diff
@@ -214,8 +214,10 @@ check_is_package <- function(whos_asking = NULL) {
 
   message <- "Project {.val {project_name()}} is not an R package."
   if (!is.null(whos_asking)) {
+    # remove brackets, so that it is recognized as a help link
+    whos_asking_fn <- sub("\\(\\)", "", whos_asking)
```

Comment:
```suggestion
    whos_asking_fn <- sub("()", "", whos_asking, fixed = TRUE)
```

I avoid backslashes in regexes whenever I can.

https://xkcd.com/1638